pom.xml

```diff
@@ -483,7 +489,7 @@
         <carbon.kernel.version>4.9.17</carbon.kernel.version>
         <carbon.kernel.package.import.version.range>[4.6.0, 5.0.0)</carbon.kernel.package.import.version.range>
         <carbon.user.package.import.version.range>[1.0.1, 2.0.0)</carbon.user.package.import.version.range>
-        <carbon.identity.framework.version>5.25.509</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.0.38-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Update the version once the framework PR is merged.

Reply:
Will approve after this